src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/orders.yml

```diff
@@ -340,3 +340,16 @@ admin_orders_configure_product_pagination:
     _legacy_controller: AdminOrders
   options:
     expose: true
+
+admin_orders_display_customization_image:
+  path: /display-customization-image/{type}/{name}/{value}
+  methods: [GET]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Order/Order:displayCustomizationImage
+    _legacy_controller: AdminOrders
+  requirements:
+    type: \d+
+    name: \w+
+    value: \w+
+  options:
+    expose: true
```

Comment:
Why do `expose` it ? It won't be used in JS.

Reply:
you're right ! fixed